src/views/view_manager.cpp

```diff
@@ -65,7 +65,7 @@ void ui::ViewManager::text(const std::string& text, int32_t x, int32_t y)
   for (size_t i = 0; i < text.length(); ++i)
   {
     SDL_Rect src = { 8 * (text[i] % GLYPHS_PER_ROW), 8 * (text[i] / GLYPHS_PER_ROW), 4, 6 };
-    SDL_Rect dest = { x + 4 * i * scale, y, 4 * scale, 6 * scale };
+    SDL_Rect dest = { static_cast<int>(x + 4 * i * scale), y, static_cast<int>(4 * scale), static_cast<int>(6 * scale) };;
```

Comment:
add a `inline static SDL_Rect SDL_MakeRect(int x, int y, int w, int h) { return { x, y, w, h }; }` to SDL_Helper.h and use the function instead that casting around